### ISSUE TITLE
acc is always about 0.5 when training cp task

### ISSUE THREAD

**Issue by likeucode** [owj0421/OutfitTransformer#3]
Hi, the acc is always about 0.5 and there is no decline in training losses when i try to train a cp model. Has anything like that ever happened to you?

**Comment by bigohofone** [comment]
That's one of the reasons I posted Checkpoint. Maybe it's because of Plateu phenomenon, after 2-3 hours of vibration, loss starts to converge. For mitigation, I tried to make warm-up schedulers, optimizer changes, and so on and I was able to improve.

**Comment by bigohofone** [comment]
I've committed the refactored code to train CP, FITB task model. I haven't tested it because I don't have GPU yet, but I've done some optimization that I didn't have before, so I think there will be an improvement if you use it.

**Comment by likeucode** [comment]
Thank you for your prompt reply, i will try it, thx~

**Comment by bigohofone** [comment]
I checked my wandb and found this log! Hope this will help you.

**Comment by bigohofone** [comment]
![W B Chart 2024  2  27  오전 12_06_27](https://github.com/owj0421/OutfitTransformer/assets/98876272/6472d29b-07b5-4ba3-8546-75695ac3cba9)

**Comment by bigohofone** [comment]
After modifying the code, I updated the required checkpoints for CP and CIR. We ask for your interest!

**Comment by zztMermory** [comment]
> ![W B Chart 2024 2 27 오전 12_06_27](https://private-user-images.githubusercontent.com/98876272/307841621-6472d29b-07b5-4ba3-8546-75695ac3cba9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyNzMyMzAsIm5iZiI6MTcxODI3MjkzMCwicGF0aCI6Ii85ODg3NjI3Mi8zMDc4NDE2MjEtNjQ3MmQyOWItMDdiNS00YmEzLTg1NDYtNzU2OTVhYzNjYmE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDEwMDIxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1MjY2NWRmNjJlYjc4ZTM3Y2ZkNGI1YWJhNjk2NTY4OTQ4ZmU2ZDg1ZDdmMGRjNzkzM2RmM2E3ODU5ODhkODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6BCoa9rhn9gR2mmM2iFCy_I1_kY3BknkyX5PbwKT2js)

How many epoch had you ever trained ?

**Comment by zztMermory** [comment]
> ![W B Chart 2024 2 27 오전 12_06_27](https://private-user-images.githubusercontent.com/98876272/307841621-6472d29b-07b5-4ba3-8546-75695ac3cba9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyNzMyMzAsIm5iZiI6MTcxODI3MjkzMCwicGF0aCI6Ii85ODg3NjI3Mi8zMDc4NDE2MjEtNjQ3MmQyOWItMDdiNS00YmEzLTg1NDYtNzU2OTVhYzNjYmE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDEwMDIxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1MjY2NWRmNjJlYjc4ZTM3Y2ZkNGI1YWJhNjk2NTY4OTQ4ZmU2ZDg1ZDdmMGRjNzkzM2RmM2E3ODU5ODhkODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6BCoa9rhn9gR2mmM2iFCy_I1_kY3BknkyX5PbwKT2js)

Hi ~ How many epoches had you ever trained?

**Comment by bigohofone** [comment]
Hi. I uploaded training setting on reademe. thanks for insterest.
